plugins/repository-s3/src/main/java/org/elasticsearch/cloud/aws/blobstore/S3BlobStore.java

```diff
@@ -182,4 +189,22 @@ protected boolean shouldRetry(AmazonClientException e) {
     @Override
     public void close() {
     }
+
+    private CannedAccessControlList initCannedACL(String cannedACL){
```

Comment:
I was wondering if we can make this method `static` and may be `public` or `protected` so we can better unit test it.

One test that come to my mind is may be to iterate over the value we know (static list of what you described in the PR description).
And also iterate other the list of what `CannedAccessControlList ` can have and check if we are missing any in our static list).

So if AWS remove a feature, we will be aware of it when we upgrade AWS SDK and same if they add a new option, we will know that we have to at least document it.